src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/azuredevops/model/PropertiesCollection.java

```diff
@@ -0,0 +1,5 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model;
+
+public class PropertiesCollection {
```

Comment:
What's the intent of this class? It contains no fields or methods

Reply:
Resolve